Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerOrganisationDetailsSummary.cshtml

```diff
@@ -17,22 +18,22 @@
     }
 }
 @section BeforeContent {
-    <a href="@backLink" class="govuk-back-link">Back</a>
+    <a href="@backLink" class="govuk-back-link">@StaticTextResource.Global_Back</a>
 }
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <partial name="_ErrorSummary" model="@ModelState" />
             
             <h1 class="govuk-heading-l">
-            <span class="govuk-caption-l">Buyer details</span>
+            <span class="govuk-caption-l">@StaticTextResource.SupplierToBuyer_OrganisationSummary_BuyerDetails</span>
                 @ViewData["Title"]
             </h1>
             
                 <dl class="govuk-summary-list">
                     <div class="govuk-summary-list__row">
                         <dt class="govuk-summary-list__key">
-                            Organisation type
+                            @StaticTextResource.SupplierToBuyer_OrganisationSummary_OrganisationType
```

Comment:
Missed `Change<span class="govuk-visually-hidden"> the organisation type` on line 49